addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -509,6 +509,12 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
             },
         ];
     }
+    /**
+     * @returns {boolean} true if the page has been altered.
+     */
+    _isDirty() {
+        return this.widget.isDirty() || Object.values(this.pageOptions).some(option => option.isDirty);
```

Comment:
@detrouxdev The test added by the commit passes without the `|| Object.values(this.pageOptions).some(option => option.isDirty)` part :/ Something else to solve ?

Reply:
Interesting, without my change, I manage to reproduce the faulty behavior manually. I think this might be due to the isDirty of the wysiwyg comparing raw html values which can change on website without any sort of actual edition. (Widgets, navbar adapting to viewport size, etc).

If I instead change the _isDirty method of the wyiswyg adapter to look for .o_dirty elements, (same as it's done to find dirty elements when saving the page), the test fails every time.

I'm not sure what the best approach is as I thought that relying on the widget's method was a good idea... What do you think? Should I look for `.o_dirty` elements instead?

Reply:
> I'm not sure what the best approach is as I thought that relying on the widget's method was a good idea... 

It's a good idea, you should find out (and be sure about it :wink:) why the widget's method says the page is dirty while it is not during the test :+1: (with the help of the editor team if needed and maybe worth another test about that somewhere) :+1: